### PR TITLE
Add aria to hello world samples

### PR DESCRIPTION
**Pull request by achocron**
Add aria to targets on ionq hello world samples

### CODE REVIEW

File: samples/azure-quantum/hello-world/HW-ionq-qsharp.ipynb [hunk 1]
```diff
@@ -297,11 +298,14 @@
...
  "metadata": {
+  "interpreter": {
+   "hash": "81794d4967e6c3204c66dcd87b604927b115b27c00565d3d43f05ba2f3a2cb0d"
+  },
   "kernel_info": {
    "name": "python3"
   },
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3.9.5 64-bit",
```
Comment (xinyi-joffre) [inline samples/azure-quantum/hello-world/HW-ionq-qsharp.ipynb:308]: All of these small other changes are making me a bit nervous--can we just make the changes to the markdown text itself?

Comment (cgranade) [reply samples/azure-quantum/hello-world/HW-ionq-qsharp.ipynb:308]: This looks like the result of running in a different Python kernel, causing metadata associated with the kernel to be updated — did you run in VS Code notebooks, per chance?

Comment (achocron) [reply samples/azure-quantum/hello-world/HW-ionq-qsharp.ipynb:308]: Oh yeah that's not intentional, must be due to my environment...will update.

Comment (achocron) [reply samples/azure-quantum/hello-world/HW-ionq-qsharp.ipynb:308]: @cgranade i did edit in vscode. I suppose that's why.

**Review by xinyi-joffre — APPROVED** [review]

**Pull request merged**